src/main/java/io/token/banksample/config/ConfigParser.java

```diff
@@ -92,6 +103,79 @@ public ConfigParser(Config config) {
                 .collect(toList());
     }
 
+    /**
+     * Extracts bank id from config
+     *
+     * @return bank id
+     */
+    public String bankId() {
+        return config.getString("bank.bank-id");
+    }
+
+    /**
+     * Extracts the secret key store for generating bank authorization payload.
+     *
+     * @return SecretKeyStore
+     */
+    public SecretKeyStore secretKeyStore() {
```

Comment:
If I remembered right about the "only use this for bank acct linking plz", maybe mention that in the method name

Reply:
I wrapped the keys in account-linking in config and added "for account linking" to all the method javadocs... hopefully that's enough of a hint. If you'd like I can change the method name as well, I just thought accountLinkingSecretKeyStore, accountLinkingEncryptionId, etc. sounded a bit cumbersome

Reply:
YAY, looks good to me.